Makefile

```diff
@@ -836,7 +845,11 @@ ifeq ($(WASI_SNAPSHOT), p2)
 		> "$(SYSROOT_INC)/__wasi_snapshot.h"
 endif
```

Comment:
@sunfishcode, I actually have a Bash script that we could also use here to move out all of these commands out of the Makefile along with the list of `MUSL_OMIT_HEADERS`. Should I bring that in to this PR?

Reply:
That sounds like a good change; would you mind submitting a separate PR for that?

Reply:
#552